tests/run-tests-compose.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+
+source .env
+
+export PODMAN_IGNORE_CGROUPSV1_WARNING=1
+
+cleanup () {
+    echo "cleaning"
+    podman-compose -f tests/compose.yml down --volumes --remove-orphans --timeout 0
+    echo "podman environment cleaned"
+}
+
+# cleanup
+
+# debug
+echo "RUNNING CONTAINERS:"
+podman ps --filter "name=vault-manager-test"
+
+# populate necessary vault access vars to primary
+podman-compose -f tests/compose.yml exec primary-vault vault kv put -mount=secret master rootToken=root
+podman-compose -f tests/compose.yml exec primary-vault vault kv put -mount=secret oidc client-secret=my-special-client-secret
+podman-compose -f tests/compose.yml exec primary-vault vault kv put -mount=secret kubernetes cert=very-valid-cert
+
+# populate oidc client secret in secondary
+podman-compose -f tests/compose.yml exec secondary-vault vault kv put -mount=secret oidc client-secret=my-special-client-secret
+podman-compose -f tests/compose.yml exec secondary-vault vault kv put -mount=secret kubernetes cert=very-valid-cert
+
+# run test suit
+for test in $(find tests/bats/ -type f | grep .bats | grep -v roles | grep -v entities | grep -v groups | grep -v errors); do
```

Comment:
Maybe simpler to read

```suggestion
for test in $(find tests/bats/ -type f | grep '\.bats$' | grep -vE 'roles|entities|group|errors'); do
```

Can you also add a comment on why you're excluding certain bats files?

Reply:
Looking into this.

Reply:
I still see the old syntax :(

Reply:
[dbd908574861cf40ea6847123f8ec3f787d82a9e](https://github.com/app-sre/vault-manager/pull/135/commits/c3e0e6fc76d6d62ce3982be7a8dfa871d7d0d06f)

I added a line here to describe why it is performed in this order.